src/commcare_cloud/ansible/roles/edit/tasks/main.yml

```diff
@@ -3,9 +3,9 @@
   become: yes
   apt: name=vim state=present
 
-- name: Install Emacs
-  become: yes
-  apt: name=emacs23 state=present
+#- name: Install Emacs
+#  become: yes
+#  apt: name=emacs23 state=present
```

Comment:
was this meant to be commented or just deleted?

Reply:
Yeah I guess deleted is better.